actionpack/test/dispatch/mime_type_test.rb

```diff
@@ -51,7 +51,7 @@ class MimeTypeTest < ActiveSupport::TestCase
     accept = "application/*"
     expect = [Mime[:html], Mime[:js], Mime[:xml], Mime[:rss], Mime[:atom], Mime[:yaml], Mime[:url_encoded_form], Mime[:json], Mime[:pdf], Mime[:zip], Mime[:gzip]]
     parsed = Mime::Type.parse(accept)
-    assert_equal expect, parsed
+    assert_equal parsed | expect, parsed
```

Comment:
This test was breaking. The ordering of the mime types was different between expected and parsed, but the union of the two shows they contain the same elements.